cbcontainers/communication/gateway/api_gateway.go

```diff
@@ -4,10 +4,16 @@ import (
 	"crypto/rsa"
 	"crypto/tls"
 	"crypto/x509"
+	"errors"
 	"fmt"
+	"net/http"
+
 	"github.com/go-resty/resty/v2"
 	"github.com/vmware/cbcontainers-operator/cbcontainers/models"
-	"net/http"
+)
+
+var (
+	ErrGettingCompatibilityMatrix = errors.New("error while getting compatibility matrix")
```

Comment:
I think the error can be change a bit cause we are not getting the matrix anymore